WebRoot/js/zimbraAdmin/resource/view/ZaNewResourceXWizard.js

```diff
@@ -554,9 +554,13 @@ ZaNewResourceXWizard.myXFormModifier = function(xFormObject) {
                 addButtonLabel:ZaMsg.NAD_AddSignature, removeButtonLabel: ZaMsg.NAD_RemoveSignature,  showAddOnNextRow:true,
                 addButtonCSSStyle:"margin-left:200px",
 				showAddButton:true, showRemoveButton:true,
+                visibilityChecks: [],
+                enableDisableChecks: [],
                 items: [
                     {
-                        ref:".", type:_SIGNATURE_, width:"100%"
+                        ref:".", type:_SIGNATURE_, width:"100%",
+                        visibilityChecks: [],
+                        enableDisableChecks: [ZaResourceXFormView.isSignagureSectionVisible]
```

Comment:
should this be part of visibilityChecks?

Reply:
Fixed it.
Moved it to another visibilityChecks for the parent item so that not only Name and Content fields but also "Add Signature" button is shown/hidden.